lib/compress.js

```diff
@@ -176,6 +176,21 @@ merge(Compressor.prototype, {
         }
     };
 
+    // we shouldn't compress (1,func)(something) to
+    // func(something) because that changes the meaning of
+    // the func (becomes lexical instead of global).
+    function check_function_call(parent, orig, val) {
```

Comment:
@alexlamsl That name works for me. Also, you are correct that parentheses don't change the Reference in any way, so they don't have an effect here.